spec/models/profile_spec.rb

```diff
@@ -23,8 +23,8 @@
       profile.encrypt_pii(pii, user.password)
 
       expect(profile.encrypted_pii).to_not be_nil
-      expect(profile.encrypted_pii).to_not match 'Jane'
-      expect(profile.encrypted_pii).to_not match '666'
+      expect(profile.encrypted_pii).to_not eq 'Jane'
+      expect(profile.encrypted_pii).to_not eq '666'
```

Comment:
Maybe. I know this is one of those tests that fails occasionally, but we're not just checking that the encrypted pii doesn't equal the unencrypted (simply JSON encoding the bundle of pii will pass the new tests).